### PR TITLE
TST: Fix scale in test

### PR DESCRIPTION
**Pull request by bashtage**
Fix exogenous scaling to improve numerical stability of test

Failing on Win/64 with OpenBLAS

### CODE REVIEW

File: statsmodels/tsa/tests/test_arima.py [hunk 1]
```diff
@@ -2107,57 +2107,50 @@ def test_arima_diff2():
...
-             9.4,   9.5,   9.5,   9.6,   9.8,  10. ,   9.9,   9.9,   9.7,
-             9.8,   9.9,   9.9,   9.6,   9.4,   9.5,   9.5,   9.5,   9.5,
-             9.8,   9.4,   9.1,   9. ,   9. ,   9.1,   9. ,   9.1,   9. ,
-             9. ,   9. ,   8.8,   8.6,   8.5,   8.2,   8.3,   8.2,   8.2,
-             8.2,   8.2,   8.2,   8.1,   7.8,   7.8,   7.8,   7.9,   7.9,
-             7.7,   7.5,   7.5,   7.5,   7.5,   7.3,   7.2,   7.2,   7.2,
-             7. ,   6.7,   6.6,   6.7,   6.7,   6.3,   6.3]
-
-    # rescaling results in convergence failure
-    #model = sm.tsa.ARIMA(np.array(ef)*100, (1,1,1), exog=ue)
```
Comment (jbrockmendel) [inline statsmodels/tsa/tests/test_arima.py]: this line needed?

File: statsmodels/tsa/tests/test_arima.py [hunk 2]
```diff
@@ -2107,57 +2107,50 @@ def test_arima_diff2():
...
+                           exog=ue[-11:], typ='levels')
 
     # regression test, not verified numbers
-    # if exog=ue in predict, which values are used ?
-    predicts_res = np.array(
-          [ 0.02612291,  0.02361929,  0.024966  ,  0.02448193,  0.0248772 ,
-            0.0248762 ,  0.02506319,  0.02516542,  0.02531214,  0.02544654,
-            0.02559099,  0.02550931])
-
     # if exog=ue[-11:] in predict
```
Comment (jbrockmendel) [inline statsmodels/tsa/tests/test_arima.py]: ditto

**Comment by josef-pkt** [comment]
(I hope I never have to look at that unit test again.)

merge when green

**Comment by bashtage** [comment]
@josef-pkt I think it is green enough (half).

**Comment by josef-pkt** [comment]
Given that this was one of the problematic test cases, I'd rather wait for appveyor results (it's next in queue).

**Comment by josef-pkt** [comment]
all green now, merged

@bashtage thank you

**Pull request merged**